cloudfunctions/candidate_gmail_messages/cloudfunction.go

```diff
@@ -344,11 +350,67 @@ func (cf *CloudFunction) proccessMessages(messageIDs []string) error {
 
 	// Get IDs of new recruiting emails
 	recruitingEmailIDs := []string{}
-	for id, result := range results.Results {
-		if !result {
+	for id, isRecruitingEmail := range results.Results {
+		// ignore non-recruting emails based on classification
+		if !isRecruitingEmail {
 			continue
 		}
 		recruitingEmailIDs = append(recruitingEmailIDs, id)
+		message := messages[id]
+		parseRequest := ml.ParseJobRequest{
```

Comment:
Can you move all of this logic into a helper function?

Reply:
✔️